frontend/src/components/Forms/Fields/Select/index.tsx

```diff
@@ -54,7 +54,18 @@ export const Select: FC<IPickerProps> = ({
       }}
       multiple={multiple}
       value={fieldValue}
-      renderOption={(props, option) => <RenderOption key={option?.value} option={option} />}
+      renderOption={(props: object, option: IOption) => {
+        return (
+          <Box
+            component="li"
+            sx={{ background: option?.option_style }}
```

Comment:
@ASpiteri-BCGov I think it was just missing its other props, like in the mui 'Country select' demo at https://mui.com/material-ui/react-autocomplete/#country-select